autopush-common/src/notification.rs

```diff
@@ -47,6 +47,9 @@ impl Notification {
     ///     {chid}:{message_id}
     pub fn chidmessageid(&self) -> String {
         let chid = self.channel_id.as_hyphenated();
+        #[cfg(feature = "redis")]
+        return format!("{}:{}", chid, self.version);
+
         if let Some(ref topic) = self.topic {
```

Comment:
I'd be surprised if clippy doesn't complain about this. 
Might want to add a feature flag before this line for `#[cfg(feature = "bigtable")] or I guess `#[cfg(not(feature = "redis"))]`

Honestly, you could probably use this untouched for the redis key to make the Topic messages work correctly.

Reply:
It works well without touching this function. It actually introduced a big if it was compiled with the 2 features